jobs/rocoto/globus_earc.sh

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+
+###############################################################
+# Source FV3GFS workflow modules
+. "${HOMEgfs}"/ush/load_fv3gfs_modules.sh
+status=$?
+[[ ${status} -ne 0 ]] && exit "${status}"
```

Comment:
```suggestion
. "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
status=$?
if [[ ${status} -ne 0 ]]; then exit "${status}"; fi
```
In a previous PR, @DavidHuber-NOAA explicitly made these if-then-else

Reply:
Yep, at my own suggestion.